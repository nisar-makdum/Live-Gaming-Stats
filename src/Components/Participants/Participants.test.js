import { render, screen, cleanup } from '@testing-library/react'
import renderer from 'react-test-renderer'
import GetParticipants from "./Participants"


afterEach(() => {
    cleanup()
})

test('test', () => {
    expect(true).toBe(true)
})

test('it will render GetParticipants component', () => {
    render(<GetParticipants />)
    const GetParticipantsElement = screen.getByTestId('test-GetParticipants')
    expect(GetParticipantsElement).toBeInTheDocument()
})

test('matches snapshot for GetCommunityImages', () => {
    const tree = renderer.create(<GetParticipants />).toJSON()
    expect(tree).toMatchSnapshot()
})