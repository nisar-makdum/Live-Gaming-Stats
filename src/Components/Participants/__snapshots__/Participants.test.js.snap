// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for GetCommunityImages 1`] = `
<div
  className="container"
  data-testid="test-GetParticipants"
>
  <div
    className="row"
  >
    <div
      className="col-md-8 p-0"
    >
      <table
        className="table mt-5 text-white border"
      >
        <thead>
          <tr>
            <th
              className="text-light text-left"
              scope="col"
            >
              Sr No
            </th>
            <th
              className="text-light"
              scope="col"
            >
              Name
            </th>
            <th
              className="text-light"
              scope="col"
            >
              Played/Won
            </th>
            <th
              className="text-light"
              scope="col"
            >
              Info
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
    </div>
    <div
      className="col-md-4 mt-5 p-0"
    >
      <div
        class="card bg-dark"
        id="card"
        style={
          Object {
            "display": "none",
            "marginTop": "120px",
          }
        }
      >
        <div
          class="card-body "
        >
          <p
            class="card-text text-info text-bold py-0 my-0 fw-bolder"
          >
            Winner
          </p>
          <p
            class="card-text text-info fw-bold"
            id="fullName"
          />
          <p
            class="card-text  fw-bold"
            id="totalPlayed"
          />
          <table
            className="table-dark container"
          >
            <tr>
              <th
                className="text-danger"
              >
                Loser
              </th>
            </tr>
            <tr>
              <td
                className="text-danger"
                id="loserOne"
              />
            </tr>
            <tr>
              <td
                className="text-danger"
                id="loserTwo"
              />
            </tr>
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;
